src/Compilers/CSharp/Portable/CodeGen/CodeGenerator.cs

```diff
@@ -137,6 +137,11 @@ private bool IsDebugPlus()
             return _module.Compilation.Options.DebugPlusMode;
         }
 
+        private bool IsPEVerifyCompatible()
```

Comment:
I do not have much preferences here.

---
In reply to: [131440368](https://github.com/dotnet/roslyn/pull/21269#discussion_r131440368) [](ancestors = 131440368)